dvc/ignore.py

```diff
@@ -222,7 +240,8 @@ def _update(self, dirname):
     def _update_sub_repo(self, root, dirs):
         for d in dirs:
             if self._is_dvc_repo(root, d):
-                new_pattern = DvcIgnorePatterns(["/{}/".format(d)], root)
+                self._ignored_subrepos.add(f"{root}{os.sep}{d}")
+                new_pattern = DvcIgnorePatterns([f"/{d}/"], root)
```

Comment:
I know that fstrings are better than format() , but there wasn't really a reason to change it in this PR :slightly_smiling_face:

Reply:
I had to change that before when I was experimenting with it. And, f-string is easy to read and understand (here, speed does not matter but could have been another reason), and therefore, I kept it.